### PR TITLE
small change in one header plus on Security page

### PR DESCRIPTION
**Pull request by jhdalek55**
There were one or two sentences where the language was almost completely redundant. I tried to simplify those without changing the meaning. Under the dependency and conflicts section, I also modified the subheads because a subhead should not have the exact same title as a heading.

### CODE REVIEW

**Comment by jhdalek55** [comment]
I have finished reading through the deployment pages now. I will try to wrap up my review of the Standard tomorrow. Thanks.
Lois

File: security_considerations.md [hunk 1]
```diff
@@ -20,23 +20,23 @@ In order to avoid this problem, the custom Targets metadata about unencrypted im
...
 
-In order to mitigate this problem, it is RECOMMENDED that a Primary use a broadcast network such as CAN, CAN FD, or Ethernet to transmit metadata to all of its Secondaries at the same time. Note that this still does not guarantee that ECUs will always see the same versions of metadata at any time. This is because network traffic between Primaries and Secondaries may still get disrupted, especially if they are connected through intermediaries, such as gateways. Nevertheless, it should still be better than unicasting.
+In order to mitigate this problem, it is RECOMMENDED that a Primary use a broadcast network, such as CAN, CAN FD, or Ethernet to transmit metadata to all of its Secondaries at the same time. Note that this still does not guarantee that ECUs will always see the same versions of metadata at any time. This is because network traffic between Primaries and Secondaries may still get disrupted, especially if they are connected through intermediaries, such as gateways. Nevertheless, it should still be better than unicasting.
```
Comment (patrickvacek) [inline security_considerations.md:33]: Wouldn't this require another comma after "Ethernet"?

File: security_considerations.md [hunk 2]
```diff
@@ -20,23 +20,23 @@ In order to avoid this problem, the custom Targets metadata about unencrypted im
...
 
-### Managing dependencies and conflicts between ECUs
+### Dependencies and conflicts between ECUs
 
-The *dependencies* for a given ECU is the set of other images that SHOULD also be installed on other ECUs in order for that image to work on a particular ECU. The *conflicts* for the same image and ECU is the set of other images that SHOULD NOT be installed on other ECUs if the ECU in question is to work correctly. *Dependency resolution* is the process of determining which versions of the latest images and their dependencies can be installed without conflicts.
+When installing an image on any given ECU, there will be *dependencies*, or a set of other images that SHOULD also be installed on other ECUs in order for the image to work. Likewise, the same image and ECU will have *conflicts*,  or a set of other images that SHOULD NOT be installed on other ECUs. *Dependency resolution* is the process of determining which versions of the latest images and their dependencies can be installed without conflicts.
```
Comment (patrickvacek) [inline security_considerations.md:39]:
```suggestion
When installing an image on any given ECU, there may be *dependencies*, or a set of other images that SHOULD also be installed on other ECUs in order for the image to work. Likewise, the same image and ECU may have *conflicts*, or a set of other images that SHOULD NOT be installed on other ECUs. *Dependency resolution* is the process of determining which versions of the latest images and their dependencies can be installed without conflicts.
```
Seems to me like both cases are not givens, but rather potentialities.

**Review by patrickvacek — COMMENTED** [review]
Looks good with just a couple minor suggestions.

File: security_considerations.md (hunk 1 above)
Comment (jhdalek55) [reply security_considerations.md:33]: Yes, to be consistent with Oxford comma.

**Pull request merged**